move_failed_tasks_back.py

```diff
@@ -26,3 +39,11 @@
 
 for task_file in failed_task_files:
     move(task_file.as_posix(), (Path.cwd() / "celery" / "out").as_posix())
+
+# remove errant messages
```

Comment:
remove the message that we are attempting to retry so that we don't retry it again.